comments-bundle/contao/dca/tl_comments.php

```diff
@@ -390,13 +385,15 @@ protected function isAllowedToEditComment($intParent, $strSource)
 		// Order deny,allow
 		$cache[$strKey] = false;
 		$security = System::getContainer()->get('security.helper');
+		$db = Database::getInstance();
```

Comment:
In this case the `$db` variable also seems to not get reused as every `case` only uses it once.

Reply:
Changed in 21fdc957ab098dba352f984b67bb8c4546d3eb05.